src/components/Single-Article.jsx

```diff
@@ -34,9 +39,21 @@ export const SingleArticle = () => {
                 <p>Comments: {article[0].comment_count}</p>
```

Comment:
consider extracting the logic for the comments out to its own component to reduce the code within this component and have more of a separation of concerns here